# -*- coding: utf-8 -*-

def error_log(error):
	import sys
	print("An error occured! \n{0}".format(error))
	sys.exit(1)
